civiform/civiform
# Add program image alt text to admin translations page.

### Description

Updates the admin translations page to include image description translations if an image description is set.

If there's no image description set, then image description isn't included in the translations page at all.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


### Instructions for manual testing

1. Edit a program > Click "Edit program image" > provide an image description.
2. Edit a program's translations > Verify there's a new "Program image description" field shown for each language on the translations page.
3. Verify you can save translated image descriptions for each language.

### Issue(s)
Fixes #5998 

Epic: #5676
